### PR TITLE
Add tests for items encoding

### PR DESCRIPTION
**Pull request by kyleconroy**

### CODE REVIEW

File: lib/StripeMethod.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ function stripeMethod(spec) {
                   : utils.makeURLInterpolator(spec.path || '');
   var requestMethod = (spec.method || 'GET').toUpperCase();
   var urlParams = spec.urlParams || [];
+  var encode = spec.encode || function(data) {return data;};
```
Comment (brandur) [inline lib/StripeMethod.js]: Mind documenting this with the other parameters above?

File: lib/resources/Subscriptions.js [hunk 2]
```diff
@@ -3,15 +3,37 @@
 var StripeResource = require('../StripeResource');
 var stripeMethod = StripeResource.method;
 
+function encode(data) {
+  if (Array.isArray(data.items)) {
+    var items = {};
+    data.items.map(function(item, i) {
```
Comment (brandur) [inline lib/resources/Subscriptions.js]: Can we put this in `utils` under `arrayToObject` a little like in Ruby? (and with a test)

File: lib/StripeMethod.js (hunk 1 above)
Comment (kyleconroy) [inline lib/StripeMethod.js]: Fixed

File: lib/resources/Subscriptions.js (hunk 2 above)
Comment (kyleconroy) [inline lib/resources/Subscriptions.js]: Fixed

**Comment by kyleconroy** [comment]
@brandur can you take a second look?

**Comment by brandur** [comment]
@kyleconroy Thanks!

LGTM.

**Pull request merged**